### DISCUSSION TITLE
How does one get Touch support with a display that is running in 8 bit parallel mode?

### DISCUSSION THREAD

**Discussion by MartinMueller2003** [Bodmer/TFT_eSPI#2170] [Q&A - Touch]
I have a  ILI9486 based TFT display (with touch) but the driver clearly states that it does not support touch functionality while in 8 bit parallel mode. How do I get touch support? A different library? If so, which library?

**Comment by Bodmer** [comment]
See the end of the ReadMe section [here](https://github.com/Bodmer/TFT_eSPI#8-bit-parallel-support) for a link to a suitable library.
